librad/src/identities/git.rs

```diff
@@ -546,6 +572,17 @@ impl<'a> Identities<'a, Project> {
             .verified(parent.as_ref())?)
     }
 
+    pub fn is_fork(&self, left: &Project, right: &Project) -> Result<bool, error::Store> {
```

Comment:
Wondering if this needs to be monomorphic in `Person` / `Project` -- there is an implied invariant which is that `left` and `right` are actually `VerifiedIdentity<T>`.